File: Week 1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int i,j,k;
    printf("Enter the height: ");
    int n = get_int();
    printf("Height: %i\n",n);
    if(n>=1)
    {
        for (i=1;i<=n;i++)
        {
            for(j=n;j>i;j--)
                printf(" ");
            for(k=1;k<=i;k++)
            {
                printf("#");
            }
            printf("\n");
        }
    }
    else
    {
        printf("Enter a suitable integer value!");
    }
}

File: Week 8/mario.py
import sys
import cs50

print ("Enter the Height--")
n=cs50.get_int()
k = 30
for i in range(0, n):
	for j in range(0, k):
		print(end=" ")
	k = k - 2
	for j in range(0, i+1):
		print("# ", end="")
	print()
